transaction-service/src/main/kotlin/ua/com/foxminded/restClient/nats/NatsController.kt

```diff
@@ -33,6 +33,7 @@ class NatsController @Autowired constructor(
 
 
     fun handleMessage(message: Message) {
+        println(message.data)
```

Comment:
please don't commit printlns. if you actually want to log something, you can use Logger

Reply:
ok